packages/server/src/http.ts

```diff
@@ -0,0 +1,49 @@
+import { Deferred, Operation, Task, once } from 'effection';
+import { Express } from 'express';
+import type { AddressInfo } from 'net';
+
+export type { AddressInfo } from 'net';
+
+export interface Server {
+  listening(): Operation<AddressInfo>;
```

Comment:
`listening` sounds like it should return a `boolean` but instead returns `AddressInfo`.

Reply:
Hmmm, what do you think is a good name? I didn't want to call it `listen()` since that would imply it is at that point that we're binding the port, but the chances are good that the port is already bound when you call that method.

Reply:
> Hmmm, what do you think is a good name? I didn't want to call it `listen()` since that would imply it is at that point that we're binding the port, but the chances are good that the port is already bound when you call that method.

Might be just my opinion.  Happy to go with `listening` for now